cmake/configs/posix_rpi_common.cmake

```diff
@@ -74,6 +74,7 @@ set(config_module_list
 	#
 	# PX4 drivers
 	#
+	drivers/linux_sbus
```

Comment:
Ok.  
1.  I've tested it on raspberry pi and  this driver is only working under Linux.
2. Onboard serial, usbtty, and other types of serial ports can communicate with sbus receivers.

3. Assemble the components according to the circuit diagram in the following url
Http://www.playuav.com/uploads/article/20160310/56cf0f65bb1f7437c1618041a30dc308.png